heat/core/operations.py

```diff
@@ -519,10 +527,11 @@ def __reduce_op(x, partial_op, reduction_op, axis, out):
 
     if axis is None:
         partial = partial_op(x._tensor__array).reshape((1,))
-        output_shape = (1,)
+        output_shape = (1,) if keepdim else ()
```

Comment:
I do not understand this line. Could you please explain it to me? I assume it is for scalars. Should the output shape not remain to be (1,) as it is exactly one element in one dimension?

Reply:
Yes it's a scalar, that's the shape of a numpy reduction when axis=None, but of course numpy does output scalars. I suppose since a heat reduction outputs a tensor, we could stick to (1,). Is that your preference?